nrm_django/grants/admin.py

```diff
@@ -39,131 +39,157 @@ class GrantAdmin(admin.ModelAdmin):
         "created_in_instance",
         "modified_date",
         "status",
+        "wppp_status",
     ]
-    fieldsets = (
+    fieldsets = [
         (
-            "Required",
+            "",
             {
                 "fields": (
-                    ("applicant_name", "proj_cfda_no"),
-                    ("application_type", "app_submission_type"),
-                    ("app_submit_date", "app_received_date"),
+                    "proj_title",
                     ("proposed_start_date", "proposed_end_date"),
-                    ("proj_title", "status"),
-                    "progrm_responsibility_type",
-                )
-            },
-        ),
-        (
-            "Details",
-            {
-                "fields": (
-                    ("gid",),
-                    ("proj_desc", "areas_effected"),
+                    ("applicant_name",),
+                    # Org goes here
+                    ("progrm_responsibility_type",),
+                    # program categories
```

Comment:
@neilmb , do you have any idea where this one might be tucked away? I'm not seeing it on grants, and per @cmajel 's research, it should be required.

Reply:
These are in a separate table: II_GA_IP_CATEGORIES that we will need to import and hook up.

Reply:
Logged https://github.com/18F/NRM-Grants-Agreements/issues/182 to make that happen ASAP.